tox.ini

```diff
@@ -89,6 +89,7 @@ allowlist_externals =
 commands =
     sphinx-build {posargs:-E} -W -b html docs/src docs/build/html
     bash -c "set -e && cd {toxinidir}/examples && bash usage.sh"
+    bash -c "set -e && cd {toxinidir}/examples/ase && python run_ase.py"
```

Comment:
Good for now, but we should think of a way run these files. Maybe we can loop over and run only files that are executable?

Reply:
I'll try to write one (or two) loop that executes all .sh and .py scripts